ext/sqlite3/sqlite3.c

```diff
@@ -1028,6 +1025,14 @@ PHP_METHOD(SQLite3, createAggregate)
 	}
 	efree(func);
 
+	error:
+	if (ZEND_FCC_INITIALIZED(step_fcc)) {
+		zend_release_fcall_info_cache(&step_fcc);
+	}
+	if (ZEND_FCC_INITIALIZED(fini_fcc)) {
+		zend_release_fcall_info_cache(&fini_fcc);
+	}
```

Comment:
`ZEND_FCC_INITIALIZED` is redundant because `zend_release_fcall_info_cache` is doing almost the same thing as checking `ZEND_FCC_INITIALIZED`.

Also, in cases like `createAggregate(null, null, null)`, there is a possibility that uninitialized members (`step_fcc.function_handler`, `fini_fcc.function_handler`) may be accessed, so it may be a good idea to set them to `NULL` in advance.
(In a follow-up PR for pdo_sqlite, I ran into this problem in the release build)